spec.md

```diff
@@ -1200,10 +1279,11 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. |
-| syslog_drain_url | string | A URL to which logs MUST be streamed. `"requires":["syslog_drain"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform MUST consider the response invalid. |
-| route_service_url | string | A URL to which the Platform MUST proxy requests for the address sent with `bind_resource.route` in the request body. `"requires":["route_forwarding"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
-| volume_mounts | array-of-objects | An array of configuration for remote storage devices to be mounted into an application container filesystem. `"requires":["volume_mount"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
+| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. MUST be returned for a synchronous request if the Service Broker supports generation of credentials. |
+| syslog_drain_url | string | A URL to which logs MUST be streamed. If the service declares `"requires":["syslog_drain"]` in the [Catalog](#catalog-management) endpoint, this MUST be returned for a synchronous request or the Platform can consider the response invalid. |
+| route_service_url | string | A URL to which the Platform MUST proxy requests for the address sent with `bind_resource.route` in the request body. If the service declares `"requires":["route_forwarding"]` in the [Catalog](#catalog-management) endpoint, this MUST be returned for a synchronous request or the Platform can consider the response invalid. |
```

Comment:
@duglin @pmorie etc Are you happy with the MUST in this description?

Before this change, we said that:
* if a broker returns `route_service_url`, then it MUST declare `"requires":["volume_mount"]` in its catalog

After this change, we say that:
* if a broker delcares `"requires":["volume_mount"]` in its catalog, it MUST return `route_service_url`

The same applies for `syslog_drain_url` and `volume_mounts`. Is everyone happy with this?

cc @jenspinney @samze

Reply:
I think the desired intent is ok but the wording is a bit off. I think its trying to say:
- - - 
A Service Broker MUST NOT return a `route_service_url` unless the corresponding
service declares `"requires":["route_forwarding"]` in the [Catalog](#catalog-management)
endpoint and the Platform has requested it by including a `bind_resource.route`
in the request body.
- - - 
Or something like that.  Right?

Reply:
I think I added something close to this on the latest commit. See what you think...